pkgs/development/python-modules/python-crontab/default.nix

```diff
@@ -37,6 +37,8 @@ buildPythonPackage rec {
     "test_03_usage"
     # AssertionError: 4 != 0
     "test_19_frequency_at_month"
+    # AssertionError: 48 != 24
```

Comment:
The reason this fails is because test is assuming $CURRENT_YEAR is not a leap year

Maybe add that to the description?

Reply:
Comment updated, thanks.